plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -139,11 +139,14 @@ private synchronized void checkPluginsForChangeAndRefresh() {
 
             for (Class<?> e : Index.list(Symbol.class, pluginManager.uberClassLoader, Class.class)) {
                 if (Descriptor.class.isAssignableFrom(e)) {
+                    Descriptor<?> d = jenkins.getDescriptorByType(e.asSubclass(Descriptor.class));
+                    if (d == null) {
+                        continue;
```

Comment:
Maybe worth logging at `Level.FINE` that the descriptor is not registered as an extension?